project_code/app1/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+const routes: Routes = [{ path: 'sample', loadChildren: () => import('./sample/sample.module').then(m => m.SampleModule) }];
```

Comment:
Change the routes with users or project, don't use samples.

Reply:
have changed the routes with user instead of sample